packages/cozy-client/src/StackLink.js

```diff
@@ -89,19 +89,19 @@ export default class StackLink extends CozyLink {
     this.stackClient = null
   }
 
-  async request(operation, result, forward) {
-    if (this.isOnline && !(await this.isOnline())) {
-      return forward(operation)
+  async request(operation, options, result, forward) {
```

Comment:
No jsdoc for the `options` ?

Reply:
Fixed [here](https://github.com/cozy/cozy-client/compare/8007276a33cdecf58aeb6e10145fcbdd09c604e0..d1b66bb18c0bdf0214f18fc8a3d3f3e7439fce4d)